src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -841,6 +841,10 @@ private ProfiledLocation ReadProfiledLocation()
 
         private EvaluationLocation ReadEvaluationLocation()
         {
+            var id = ReadInt32();
+            int? parentId = null;
+            var hasParent = ReadBoolean();
+            if (hasParent) parentId = ReadInt32();
```

Comment:
nit: add curlies for single line statement blocks

Reply:
I can change this, but I copied the style from below (look at hasLine). I thought that was the coding convention?

---
In reply to: [157254270](https://github.com/Microsoft/msbuild/pull/2813#discussion_r157254270) [](ancestors = 157254270)

Reply:
There's more curlied single lines in this file than non-curlied single lines. Curlied lines wins :)

---
In reply to: [157256344](https://github.com/Microsoft/msbuild/pull/2813#discussion_r157256344) [](ancestors = 157256344,157254270)